source/class/qx/util/PropertyUtil.js

```diff
@@ -187,8 +199,8 @@ qx.Class.define("qx.util.PropertyUtil", {
      * @param value {var} The value to set
      */
     setThemed(object, propertyName, value) {
-      var styler = qx.core.Property.$$method.setThemed;
-      object[styler[propertyName]](value);
+      let             propertyFirstUp = qx.Bootstrap.firstUp(propertyName);
```

Comment:
why space between two words so long?

Reply:
My personal style is to align all variable names in declarations at the top of the function, indented so that the length of the "type" of the variable doesn't affect the indentation of all of the variable names. Granted in JS the "type" doesn't exist, and either `var` or `let` consumes the same amount of space in all cases. It's just habit. I removed the extra indentation in the places you identified.